templates/terraform/environments/shared/main.tf

```diff
@@ -174,5 +174,5 @@ output "ci_user_name" {
 }
 
 output "cloudtrail_bucket_id" {
-  value = module.cloudtrail.cloudtrail_bucket_id
+  value = local.enable_cloudtrail > 0 ? module.cloudtrail[0].cloudtrail_bucket_id : ""
```

Comment:
This would fail later when this value is used since it will result in an invalid ARN in IAM. The output would need to be something non-empty. Maybe just `not-applicable` or something.

Reply:
will it be used if `enable_cloudtrail` is disabled ? I used it with disable without issue.